### PR TITLE
Add game page html and styles

### LINKED ISSUE #4: add style and content for game.html

### PR DESCRIPTION
**Pull request by mccoydidericksen**

### CODE REVIEW

**Comment by IanSSenne** [comment]
relates to #4

File: game.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" class="vh-100">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script src="https://cdn.tailwindcss.com"></script>
+        <script>
+            tailwind.config = {
```
Comment (IanSSenne) [inline game.html:10]: looks good, we should prob move this to the shared js file ito keep our breakpoints consistent

File: game.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          </script>
+		<title>Jeapordy Game</title>
+	</head>
+    <body class="bg-gray-800">
+        <div class="container mx-auto">
+                    <div id="game-grid" class="flex md:flex-row sm:flex-col items-center justify-center">
+                        <div class="flex flex-col items-center justify-center">
+                            <iframe src="https://giphy.com/embed/JtM4TBYqkCIAoIzsqt" class="w-1/2" alt="trivia sticker"></iframe>
+                            <p class="bg-gray-500 text-white p-2 m-4 rounded-md">Category1</p>
+                            <!-- below is an example of a selected button - change background color to bg-gray-700 and remove hover:bg-sky-700 class -->
```
Comment (IanSSenne) [inline game.html:30]: thank you for making an example of a disabled or used button

**Review by IanSSenne — APPROVED** [review]
looks good to me, suggest letting @Isglad also review before merge

File: game.html (hunk 1 above)
Comment (Isglad) [reply game.html:10]: looks good to me too. But seems like we've use tailwind which has not been approved.

**Review by Isglad — APPROVED** [review]

**Pull request merged**